soh/src/code/z_map_exp.c

```diff
@@ -751,230 +625,196 @@ void Minimap_Draw(PlayState* play) {
             Y_Margins_Minimap = 0;
         }
 
-        switch (play->sceneNum) {
```

Comment:
This diff is terrible, I am sorry. But there shouldn't be any changes other than the removal of the switches.